requirements/requirements-docs.txt

```
@@ -2,3 +2,4 @@ numpydoc>=1.1.0
 sphinx>=3.3.0
 sphinx-tabs>=1.1.13
 sphinxcontrib-bibtex==1.0.0
+elephant[extras]>=0.9.0 # needed for sphinx to create plots for GPFA and ASSET
```

I would not comment this, I think it's clear we need the extras to be able to run any function in the elephant:
```suggestion
elephant[extras]>=0.9.0
```

---

Done: 98643fde05ce270407c339c66b52ff5422ac68a3